### PR TITLE
Index + Show view updated

### PR DESCRIPTION
**Pull request by amycsong98**

### CODE REVIEW

File: marketplaceApp/app/controllers/items_controller.rb [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+    order = params[:order] || "name"
+    @items = Item.all.order(order)
   end
     
   def show
-    id = params[:id]
-    @item = Item.find(id)
+    if params[:id]
+      @item = Item.find(params[:id])
+      @seller = User.find(@item.user_id).name
```
Comment (mrubakh) [inline marketplaceApp/app/controllers/items_controller.rb:10]: I think every user is both a seller and a buyer, so should this be @user? I'm not sure how we want to pass these into the view from the controller

File: marketplaceApp/app/views/items/show.html.erb [hunk 2]
```diff
@@ -1,30 +1,31 @@
...
+    <p><%= "no" %></p>
+  <% end %>
+</div>
+<h4>Price</h4>
+<div class="item-price">
+  <p><%= number_to_currency(@item.price) %></p>
+</div>
+
+<h4>Sold by: </h4>
+<p> <%= @seller %> </p>
```
Comment (mrubakh) [inline marketplaceApp/app/views/items/show.html.erb:26]: same thing - maybe @user for consistency?

File: marketplaceApp/app/controllers/items_controller.rb (hunk 1 above)
Comment (amycsong98) [reply marketplaceApp/app/controllers/items_controller.rb:10]: yeah. i think this is probably the easiest way to pass it into the view from the controller with the instance variable.
the code that was merged yesterday to master doesn't have any rspec tests. do you guys wanna write those first before i write them for this PR?

Comment (amycsong98) [reply marketplaceApp/app/controllers/items_controller.rb:10]: yeah good point.

**Pull request merged**